adapter/sql/util.go

```diff
@@ -35,6 +40,13 @@ func Escape(config Config, field string) string {
 		return escapedField.(string)
 	}
 
+	if v, ok := isAliasField(field); ok {
+		aliasField := Escape(config, v.source) + " AS " + v.alias
```

Comment:
just follow the pattern and assign the result to `escapedField`? that way we can limit three return statement in this function (which is limited to 4 by codeclimate)

Reply:
also the, it might be better if statement below can be refactored using switch, I think it's more readable